src/test/searchqueryparsertest.cpp

```diff
@@ -382,6 +382,27 @@ TEST_F(SearchQueryParserTest, NumericFilter) {
         qPrintable(pQuery->toSql()));
 }
 
+TEST_F(SearchQueryParserTest, NumericFilterYear) {
+    QStringList searchColumns;
+    searchColumns << "year";
+
+    auto pQuery(
+            m_parser.parseQuery("year:1969", searchColumns, ""));
+
+    TrackPointer pTrack = newTestTrack(44100);
+    EXPECT_FALSE(pQuery->match(pTrack));
+    pTrack->setYear(" 1969-08-15 ");
```

Comment:
the extra spaces confuse me a little, is there intend behind that?

Reply:
This is a test that the user input is trimmed. I have added a comment.